packages/contracts/contracts/plugin/PluginSetupProcessor.sol

```diff
@@ -579,22 +570,23 @@ contract PluginSetupProcessor is DaoAuthorizable {
             _params.setupPayload
         );
 
+        // Store only the data required for subsequent updates or uninstallations and set the rest to zero.
         bytes32 newSetupId = _getSetupId(
             _params.pluginSetupRef,
             hashPermissions(permissions),
             ZERO_BYTES_HASH,
             bytes(""),
-            PreparationType.Uninstall
+            PreparationType.Uninstallation
```

Comment:
Any side effects on Subgraph/SDK?

Reply:
Subgraph is anyways broken.  I don't think SDK is up to what we're doing with the new PSP. but not sure.

Reply:
This is not reflected on the subgraph. If so, enum value didn't change so I don't think it would have an effect